src/main/org/tvrenamer/model/GlobalOverrides.java

```diff
@@ -19,7 +19,7 @@
     private Map<String, String> showNames;
 
     private GlobalOverrides() {
-        showNames = new HashMap<String, String>();
```

Comment:
It was like that when I got here!!   :-)  In fact, earlier versions of Java required the type to be re-specified.  I'm not sure when they improved it, but definitely by Java 7, so yes, we should use the cleaner syntax.  And in fact I did, in the later commit, "Infer generic types when possible".  I just left it that way in the commit you're looking at because I was being very methodical.